packages/waku/src/router/fs-router.ts

```diff
@@ -41,7 +41,7 @@ export function fsRouter(
             return [];
           }
           // HACK: replace "_slug_" to "[slug]" for build
-          file = file.replace(/(^|\/|\\)_([^/]+)_(\/|\\|\.)/g, '$1[$2]$3');
+          file = file.replace(/(?<=^|\/|\\)_([^/]+)_(?=\/|\\|\.)/g, '[$1]');
```

Comment:
Wow, nice. I should improve my regex skill.

(We need to avoid this HACK, but if it's not soon, we should probably extract the logic into a function and add a test for it.)